conan/cli/commands/graph.py

```diff
@@ -298,3 +301,109 @@ def graph_explain(conan_api, parser,  subparser, *args):
 
     ConanOutput().title("Closest binaries")
     return {"closest_binaries": pkglist.serialize()}
+
+
+def outdated_text_formatter(result):
+    cli_out_write("======== Outdated dependencies ========", fg=Color.BRIGHT_MAGENTA)
+
+    for key, value in result.items():
+        cli_out_write(value["current"].name, fg=Color.BRIGHT_YELLOW)
+        cli_out_write(f'\tCurrent: {value["current"].ref.repr_notime()}', fg=Color.BRIGHT_CYAN)
+        cli_out_write(f'\tLatest in range:  {value["latest_ref"].repr_notime()}', fg=Color.BRIGHT_CYAN)
+        cli_out_write(f'\tLatest in remote(s):  {value["latest"]["ref"].repr_notime()} - {value["latest"]["remote"].name}',
```

Comment:
I think that having the timestamp visible with `repr_humantime()` would be useful, specially for the revision -> revision case